File: LeetCode/Easy/232. Implement Queue using Stacks.py
class MyQueue:

    def __init__( self ):
        
        self.a = []
        
        

    def push( self, x: int ) -> None:
        
        self.a.append( x )
        
        

    def pop( self ) -> int:
        
        return self.a.pop( 0 )
    
    

    def peek( self ) -> int:
        
        return self.a[0]
    
    

    def empty( self ) -> bool:
        
        return len ( self.a ) == 0

File: LeetCode/Easy/232. Implement Queue using Stacks.java
class MyQueue
{
    
    Stack<Integer> stack;
    
    
    
    public MyQueue()
    {
        this.stack = new Stack<Integer>();
    }
    
    
    
    public void push( int x )
    {
        Stack<Integer> stack = new Stack<>();
        
        while( !this.stack.isEmpty() )
        {
            stack.push( this.stack.pop() );
        }
        
        this.stack.push( x );
        
        while ( !stack.isEmpty() )
        {
            this.stack.push( stack.pop() );
        }
    }
    
    
    
    public int pop()
    {
        return this.stack.pop();
    }
    
    
    
    public int peek()
    {
        return this.stack.peek();
    }
    
    
    
    public boolean empty()
    {
        return this.stack.isEmpty();
    }
    
}
